examples/ittapi/boo.cpp

```diff
@@ -0,0 +1,4 @@
+#include <ittnotify.h>
+
+int main() {
```

Comment:
could you add something that uses a variable from the ittnotify library, instead of just the header. This makes sure, that the include directory AND the library path are configured correctly

Reply:
Good point. I added some calls to ittapi here.